reflex/components/next/video.py

```diff
@@ -0,0 +1,33 @@
+"""Wrapping of the next-video component."""
+
+from typing import Optional
+
+from reflex.components.component import Component
+from reflex.vars import Var
+
+from .base import NextComponent
+
+
+class Video(NextComponent):
+    """A video component from NextJS."""
+
+    tag = "Video"
+    library = "next-video"
+    is_default = True
```

Comment:
what does this component actually do? reading through the docs here https://next-video.dev/docs

it seems like it's a video uploading and hosting service with nextjs integration. i'm not 100% sure this makes sense in reflex, especially not without the associated next.config.js

```
/** @type {import('next').NextConfig} */
const { withNextVideo } = require('next-video/process');
 
const nextConfig = {}; // Your current Next Config object
 
module.exports = withNextVideo(nextConfig);
```

I think this could be part of our asset story, particularly with the hosting service which does not have a lot of disk, but it seems to be missing parts and definitely needs documentation.